pkg/userdata/bottlerocket/template.go

```diff
@@ -0,0 +1,50 @@
+package bottlerocket
+
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/mrajashree/etcdadm-bootstrap-provider/pkg/userdata"
+	"github.com/pkg/errors"
+)
+
+const (
+	cloudConfigHeader = `## template: jinja
+#cloud-config
+`
+	// sentinelFileCommand writes a file to /run/cluster-api to signal successful Kubernetes bootstrapping in a way that
+	// works both for Linux and Windows OS.
+	sentinelFileCommand = "echo success > /run/cluster-api/bootstrap-success.complete"
```

Comment:
@mrajashree I need a bit of help to understand what this is for

Reply:
just noting here that as per our discussion we don't need this for BR :)